app-utils/busybox/autobuild/config

```diff
@@ -1,15 +1,16 @@
+# https://gitlab.archlinux.org/archlinux/packaging/packages/busybox/-/blob/cebfc6cb756e7c9f41c8105e6a93a79a72da51ba/config
```

Comment:
I think we don't want to follow the whole config from archlinux: their config enables less features as ours. I think it is better to just migrate the old config?